### DISCUSSION TITLE
Option to convert Tif to Jpg ?

### DISCUSSION THREAD

**Discussion by yoan31** [Tichau/FileConverter#202] [Ideas]
Hello !

Can we have an option to convert Tif to Jpg ?

Thanks

**Comment by moosj** [comment]
Same issue here. Despite being listed as a compatible input filetype, File Converter does not show up on the context menu when I right click TIFF files.

**Reply by jpeni** [reply]
I found that File Converter only recognizes TIFF files, not TIF files.
I hope the program is updated to also recognize TIF files.
